.github/workflows/e2e-bootstrap.yaml

```diff
@@ -6,14 +6,15 @@ on:
     branches: [ 'main', 'release/**' ]
   pull_request:
     branches: [ 'main', 'release/**' ]
+    paths-ignore: [ 'docs/**', 'rfcs/**' ]
 
 permissions:
   contents: read
 
 jobs:
   e2e-boostrap-github:
     runs-on: ubuntu-latest
-    if: (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository) && github.actor != 'dependabot[bot]'
+    if: (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository) && github.actor != 'dependabot[ bot ]'
```

Comment:
This isn't correct: https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#responding-to-events

Reply:
Yeah my IDE borked the whole YAML...